misc/images/materialized/entrypoint.sh

```diff
@@ -40,6 +40,7 @@ cockroach sql --insecure -e "SET CLUSTER SETTING sql.stats.forecasts.enabled = f
 cockroach sql --insecure -e "CREATE SCHEMA IF NOT EXISTS consensus"
 cockroach sql --insecure -e "CREATE SCHEMA IF NOT EXISTS storage"
 cockroach sql --insecure -e "CREATE SCHEMA IF NOT EXISTS adapter"
+cockroach sql --insecure -e "CREATE SCHEMA IF NOT EXISTS timestamp_oracle"
```

Comment:
bikeshed: Very much a judgement call, but personally I'd be inclined to make this shorter (one word). Maybe something like "timestamps" or "oracle". Maaaaayyyybe "tsoracle" if we think those are too ambiguous

Reply:
you mean only the schema here or the naming of the parameter and things in other places?

Reply:
Mostly the things in external systems (e.g. crdb schema)

Reply:
This one here is only used for testing, and the naming of the real one will be up to the cloud team, but I went ahead and shortened it anyways. :ok_hand: